homework_weekend6/src/homework_weekend6/task2/Main.java

```diff
@@ -0,0 +1,143 @@
+package homework_weekend6.task2;
+
+import java.awt.BorderLayout;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+
+/** Creates GUI application that represents us a window with two buttons
+* and one label. Buttons allows us to insert randomly picked names and surnames,
+* making different combinations, and showing them in the label.
+* Contains two inner classes that contain names and surnames. 
+* @author ajla
+*
+*/
+public class Main extends JFrame {
+
+	private static final long serialVersionUID = 4144316998591105774L;
+	
+	// Creating buttons and label
+	private JButton buttonName = new JButton("FirstName");
+	private JLabel fullName = new JLabel("");
+	private JButton buttonSurname = new JButton("LastName");
+	private String name = "";
+	private String surname = "";
+	
+	/**
+	 * Constructor Creates a window with belonging buttons.
+	 * Adds ActionListener to the buttons.
+	 */
+	public Main () {
+		
+		setLayout(new BorderLayout());
+
+		// Setting buttons
+		add(buttonName, BorderLayout.WEST);
+		add(fullName, BorderLayout.CENTER);
+		add(buttonSurname, BorderLayout.EAST);
+		
+		// Adding action listener to the buttons
+		buttonName.addActionListener(new Action());
+		buttonSurname.addActionListener(new Action());
+		
+		fullName.setHorizontalAlignment(JLabel.CENTER);
+		
+		// Setting the layout
+		setResizable(false);
+		setTitle("Task2");
+		setSize(500, 200);
+		setLocationRelativeTo(null);
+		setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+		setVisible(true);
+	}
+	
+	/**
+	 * Creating a class named Action that implements ActionListener interface.
+	 * It allows us to create buttons behviour. 
+	 * There are two buttons: 
+	 * buttonName - returns randomly picked name and shows it in the label,
+	 * buttonSurname - returns randomly picked surname and shows it in the label.
+	 * @author ajla
+	 *
+	 */	
+	private class Action implements ActionListener {
+
+		@Override
+		public void actionPerformed(ActionEvent e) {
+			if (e.getSource() == buttonName) {
+				name = new Names().toString();
+				fullName.setText(name + " " + surname);
+			} else if (e.getSource() == buttonSurname) {
+				surname = new Surnames().toString();
+				fullName.setText(name + " " + surname);
+			}
+		}
+	}
+	
+	/**
+	 * Inner class that represents kind of names database.
+	 * Contains an array of names.
+	 * Has two methods: generateName(), that returns randomly picked up one name, and
+	 * toString(), that prints out that randomly picked name.
+	 * @author ajla
+	 *
+	 */
+	public static class Names {
+		private static int NAMES_NUMBER = 6;
```

Comment:
This variable is not needed, you can use `names.length` to achieve same thing. If you decide to add more names to the name pool, you would not be forced to change `NAMES_NUMBER` value.